### ISSUE TITLE
Add specification for non-hardened coin_type derivation

### ISSUE THREAD

**Issue by priestc** [satoshilabs/slips#88]
This spec defines `coin_type` to use hardened derivation. The result is that this specification can't be used to share a "milti-coin master seed", which could be useful.

For instance, assume a service where the user generates a xpriv and corresponding xpub. They give the xpub to the service, and then sends the service $1000 worth of bitcoin. The service then generates an LTC address using the xpub and sends $200 to that address. It also generates a Dash address from the xpub and sends $200 to that address, and so on... The user who has the xpriv can generate the private keys to actually spend these funds.

Basically it's like a ShapeShift Prism, except using HD keys instead of a smart contract. Unfortunately, because the BIP44 specification specifies hardened derivation for coin_type, this is not possible.

There should be an extension to Bip44 (or maybe another bip) that uses the same coin index as defined in slip44, but with non-hardened derivation.

**Comment by prusnak** [comment]
Feel free to write another BIP. Extension of BIP44 would create a mess.
Also, I really do think that having coin_type as hardened derivation is important, because in case your altcoin wallet is compromised, your Bitcoin funds are protected. If it is not hardened, an attacker can traverse the tree upwards and get control of other coins' keys.
